stream-chat-android/src/main/java/com/getstream/sdk/chat/adapter/CommandMentionListItemAdapter.java

```diff
@@ -53,57 +48,18 @@ public long getItemId(int position) {
     public View getView(int position, View convertView, ViewGroup parent) {
         ViewBinding binding;
         if (convertView == null) {
-            if (isCommand) {
-                convertView = layoutInflater.inflate(R.layout.stream_item_command, null);
-                binding = StreamItemCommandBinding.bind(convertView);
-            } else {
-                convertView = layoutInflater.inflate(R.layout.stream_item_mention, null);
-                binding = StreamItemMentionBinding.bind(convertView);
-            }
+            convertView = layoutInflater.inflate(R.layout.stream_item_mention, null);
+            binding = StreamItemMentionBinding.bind(convertView);
             convertView.setTag(binding);
         } else {
             binding = (ViewBinding) convertView.getTag();
         }
-
-        if (isCommand) {
-            configCommands((StreamItemCommandBinding) binding, position);
-        } else {
-            configMentions((StreamItemMentionBinding) binding, position);
-        }
+        configMentions((StreamItemMentionBinding) binding, position);
```

Comment:
If we always have just one kind of binding used here (as changed above), we can use its type for the local `binding` variable, and there's no need to cast here